dlt-database/entity/0001-init_db/User.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, OneToMany, JoinColumn } from 'typeorm'
+
+import { Account } from './Account'
+
+@Entity('users', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class User extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({
+    name: 'gradido_id',
+    length: 36,
+    nullable: true,
```

Comment:
the gradidoId of a User is not nullable. Do you need it in DLT as nullable?

Reply:
Yes, for users with a Crypto account. I don't necessarily know the Gradido ID of those.